src/WebViewTypes.ts

```diff
@@ -24,6 +24,10 @@ type WebViewCommands =
 
 type AndroidWebViewCommands = 'clearHistory' | 'clearCache' | 'clearFormData';
 
+interface UnsupportedVersionProps {
+  device: string;
+}
+
```

Comment:
this is just b/c we are passing down a prop to be handled by the component, right?

Reply:
yes